sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/parser/SqlCreateTable.java

```diff
@@ -67,23 +72,24 @@
     this.tblProperties = tblProperties; // may be null
   }
 
+  @Override
   public List<SqlNode> getOperandList() {
-    return ImmutableNullableList.of(name, columnList, type, comment, location, tblProperties);
+    throw new UnsupportedOperationException(
+        "Getting operands CREATE TABLE is unsupported at the moment");
```

Comment:
<!--new_thread; commit:4a38a26e2bfa0631ef5426455a57802eb81e5e93; resolved:1-->
Just curious - where is it used? You can't really do much except copy the node if you don't know what kind of node it is, so just asking for my own education how Calcite (or BeamSQL) uses this.

Reply:
It's mostly used by implementations of the `SqlCall` to access their parameters. `SqlCreate` itself doesn't really need it but we might (or might not) need something like this for JDBC integration, depending on how `CREATE TABLE` parsing will be implemented there